libraries/botbuilder-adapter-tyntec-whatsapp/README.md

```diff
@@ -0,0 +1,54 @@
+# Tyntec WhatsApp Adapter
+
+A [Microsoft Bot Framework](https://www.botframework.com/) adapter for handling
+connectivity with the WhatsApp channel in tyntec Conversations API.
+
+It is a TypeScript library that allows your bots to use WhatsApp through the
+tyntec Conversations API. The adapter supports two-way (incoming and outgoing)
+messaging with templates, free-form and rich media.
+
+Look how easy it is to use:
+
+```typescript
+import axios from 'axios';
+import { TyntecWhatsAppAdapter } from '@botbuildercommunity/adapter-tyntec-whatsapp';
+
+const axiosInstance = axios.create();
+
+const adapter = new TyntecWhatsAppAdapter({
+    axiosInstance,
```

Comment:
Why is it required to pass axios?

Reply:
Basically, for easier testing/mocking requests to Tyntec API. Do you suggest an alternative?